core/storage/collection/gae_models.py

```diff
@@ -65,6 +65,8 @@ class CollectionModel(base_models.VersionedModel):
     # The version of all property blob schemas.
     schema_version = ndb.IntegerProperty(
         required=True, default=1, indexed=True)
+    # DEPRECATED in v2.4.2. Do not use.
```

Comment:
Where did this come from? I don't see it in the previous version.

Reply:
I think I saw this on my Oppia instance I think it is another field that is not properly deleted (i.e. it was removed from here previously without being removed from datastore). Could you please check on prod datastore if `CollectionModel` contains `nodes`?